gateone/applications/terminal/plugins/ssh/scripts/ssh_connect.py

```diff
@@ -751,15 +760,17 @@ def main():
                 sshfp=options.sshfp,
                 randomart=options.randomart,
                 additional_args=options.additional_args,
-                socket=options.socket
+                socket=options.socket,
+		config=options.config
```

Comment:
Same as above: The indentation doesn't seem to be correct here. Try running `flake8` over this file.